arex-agent-core/src/main/java/io/arex/agent/instrumentation/InstrumentationInstaller.java

```diff
@@ -183,17 +181,17 @@ private AgentBuilder getAgentBuilder() {
         long buildBegin = System.currentTimeMillis();
         AgentBuilder builder = new AgentBuilder.Default(
                 new ByteBuddy().with(MethodGraph.Compiler.ForDeclaredMethods.INSTANCE))
-            .enableNativeMethodPrefix("arex_")
-            .disableClassFormatChanges()
-            .ignore(new IgnoredTypesMatcher())
-            .with(new TransformListener())
-            .with(AgentBuilder.RedefinitionStrategy.RETRANSFORMATION)
-            .with(AgentBuilder.InitializationStrategy.NoOp.INSTANCE)
-            .with(AgentBuilder.TypeStrategy.Default.REBASE)
-             // https://github.com/raphw/byte-buddy/issues/1441
-            .with(AgentBuilder.DescriptionStrategy.Default.POOL_FIRST)
-            .with(AgentBuilder.LocationStrategy.ForClassLoader.STRONG
-                .withFallbackTo(ClassFileLocator.ForClassLoader.ofSystemLoader()));
+                .enableNativeMethodPrefix("arex_")
+                .disableClassFormatChanges()
+                .ignore((typeDescription, classLoader, module, classBeingRedefined, protectionDomain) -> classLoader != null && classLoader.getClass().getName().equals("io.opentelemetry.javaagent.bootstrap.AgentClassLoader"))
```

Comment:
thank for your PR !!  
`IgnoredTypesMatcher` can not remove,  
you can add to `io.arex.inst.extension.matcher.IgnoreClassloaderMatcher`,  
and suggest to make it configurable, so that in the future, if there are other agents, we can also be easily excluded:
demo:
```
-Darex.ignore.classloader=io.opentelemetry
loaderName.startsWith(System.getProperty("arex.ignore.classloader"))
```
and don't forget unit test